functional_tests/test_categories.py

```diff
@@ -52,3 +52,25 @@ def testGetCategoryByName(self):
         response = discourse_client.categories.getByName('Broncos').get_result()
 
         self.assertEqual('Broncos', response['name'])
+
+    def testGetAllCategories(self):
+        discourse_client.categories.create(
+            category_name='Broncos',
+            slug='broncos'
+        )
+
+        discourse_client.categories.create(
+            category_name='SF Giants',
+            slug='sfgiants'
+        )
+
+        discourse_client.categories.create(
+            category_name='SJ Sharks',
+            slug='sjsharks'
+        )
+
+        response = discourse_client.categories.getAllCategories().get_result()
+
+        self.assertTrue(any(category['slug'] == 'broncos' for category in response))
```

Comment:
I think there's a better way to do this -- how about `self.assertEqual({'broncos', 'sfgiants', 'sjsharks'}, {category['slug'] for category in response})`?